realm/realm-library/src/objectServer/java/io/realm/SyncUser.java

```diff
@@ -426,6 +427,68 @@ public SyncUser run() {
         }.start();
     }
 
+    /**
+     * Helper method for Admin users in order to look up a {@code SyncUser} using the identity provider and the used username.
+     * @param provider identity providers {@link io.realm.SyncCredentials.IdentityProvider}.
+     * @param providerId username or email used to create the account first time.
```

Comment:
I would expand this description to e.g. `username or email used to create the account for the first. What is needed will depend on what type of {@link SyncCredentials} was used.`